about.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;0,700;0,900;1,400;1,700;1,900&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="./about.css">
+  <title>About Us</title>
+</head>
+<body>
+  <header>
+    <nav>
```

Comment:
- [optional] great job making the requested changes. Please kindly make the mobile menu full height and fixed, so it can serve its purpose because as it is the app can still be manipulated without using it even though it has come up. 
![Screenshot (1156)_LI](https://github.com/AjroudiRida/capstone-project-freelance/assets/65298048/594612fd-2c37-41b8-b50c-4e57eed22cb0)
- [optional] please kindly leave the more option button null rather than give it a link to nowhere. This will make your work user-friendly and professional.